src/pages/blog/double-dispatch-in-c-and-ddd.md

```diff
@@ -17,12 +18,13 @@ comments: true
 share: true
 ---
 
-Double dispatch is a pattern you can use in C# to control how communication flows between two objects. A frequent use of the pattern is to pass "this" to a function on another class, allowing that class to communicate back to or manipulate the calling object instance. It can also be used to achieve polymorphic behavior. I have [a GitHub sample](https://github.com/ardalis/DoubleDispatchSamples) you can browse or download that demonstrates some of these techniques.
+A frequent use of the pattern is to pass "this" to a function on another class, allowing that class to communicate back to or manipulate the calling object instance. It can also be used to achieve polymorphic behavior. I have [a GitHub sample](https://github.com/ardalis/DoubleDispatchSamples) you can browse or download that demonstrates some of these techniques.
```

Comment:
It's ok to duplicate it in the description and the leading paragraph; folks reading the article on the web won't see the description. Keep the sentence in both places.